### DISCUSSION TITLE
Question about VAC

### DISCUSSION THREAD

**Discussion by madbanana22** [Fedoraware/Fedoraware#1139] [Q&A] [answered]
Hello, I'd like to enter the battlefield and fight bots. If I have 2 accounts in the same Steam client (main and alt) and alt gets detected, will my main get banned?

**Comment by svnomore** [comment]
if only both verified with same phone number

**Comment by ghost** [comment, marked as answer]
https://www.quora.com/If-you-have-2-Steam-accounts-1-computer-and-you-get-a-VAC-ban-on-one-of-them-does-the-other-one-get-a-VAC-ban-too
